src/Evolution/Systems/ScalarWave/System.hpp

```diff
@@ -20,7 +20,18 @@ class Variables;
 
 /*!
  * \ingroup EvolutionSystemsGroup
- * \brief Items related to evolving the scalar wave equation:
+ * \brief Items related to evolving the scalar wave equation.
+ *
+ * The equations of motion for the system augmented with constraint damping
+ * terms are given by Eq. (15), (23) and (24) of \cite Holst2004wt (setting
+ * background spacetime to Minkowskian):
+ * \f{align*}
```

Comment:
Can you add a blank dox line (` *`) before the `\f{...` and after the `\f}` so ClangFormat doesn't mess up the lines?